Library/Homebrew/extend/array.rb

```diff
@@ -1,7 +1,17 @@
-# typed: true
+# typed: strict
 # frozen_string_literal: true
 
 class Array
+  # Equal to <tt>self[1]</tt>.
+  #
+  #   %w( a b c d e ).second # => "b"
+  def second = self[1]
+
+  # Equal to <tt>self[2]</tt>.
+  #
+  #   %w( a b c d e ).third # => "c"
+  def third = self[2]
+
```

Comment:
I'm sure you will see it in the CI errors, but we do use `array.fourth` occasionally in `homebrew-cask` when a version strings contains four parts.

Reply:
I appreciate the heads up nonetheless, ty!

Reply:
Looks like we also use fifth! That I did not know.